interop/jvm/src/main/scala/scalaz/zio/interop/catz.scala

```diff
@@ -0,0 +1,65 @@
+package scalaz.zio
+package interop
+
+import cats.effect
+import cats.effect.Effect
+import cats.syntax.all._
+
+import scala.util.control.NonFatal
+
+object catz extends RTS {
+
+  implicit val catsEffectInstance: Effect[Task] = new Effect[Task] {
```

Comment:
Is it possible to do anything more than `Effect`? e.g. `Concurrent`, etc.

Reply:
Didn't try, just ported it from ioeffect. Will give it a go later and see how far I get. But i think there may be a difference in semantics that makes it hard or even impossible to implement as per cats-effect 0.10

Reply:
Hey @jdegoes this PR is taking quite long, what do you think if we merge it and I work on the remaining stuff in followup (smaller) PRs? I think it is in pretty good shape already, it has more functionality that the one provided by `ioeffect` and it's got way more tests.
I can create separate issues for the rest of the tasks such as add docs and implement `Concurrent` (what I already started)